triplets/community/arm64-android.cmake

```
@@ -2,3 +2,4 @@ set(VCPKG_TARGET_ARCHITECTURE arm64)
 set(VCPKG_CRT_LINKAGE static)
 set(VCPKG_LIBRARY_LINKAGE static)
 set(VCPKG_CMAKE_SYSTEM_NAME Android)
+set(VCPKG_MAKE_BUILD_TRIPLET aarch64-linux-android)

```

you probably wanted:
```suggestion
set(VCPKG_MAKE_BUILD_TRIPLET "--host=aarch64-linux-android")
```

---

I left that here

https://github.com/microsoft/vcpkg/pull/26570/files#diff-b92b07d5c0c681687d71463c6a368fae98e9ffedb241e2aadbde3aac688899a9R476